website/content/api-docs/auth/jwt.mdx

```diff
@@ -177,6 +177,7 @@ entities attempting to login. At least one of the bound values must be set.
   time the user was actively authenticated with the OIDC provider. If set, the `max_age` request parameter
   will be included in the authentication request. See [AuthRequest](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest)
   for additional details. Accepts an integer number of seconds, or a Go duration format string.
+- `token_policies_template_claims` `(bool: false)` - When enabled, allows token_policies to represent templates which are computed with all claims on the underlying JWT or OIDC token. This uses [`sdk/helper/template`](https://pkg.go.dev/github.com/openbao/openbao/sdk/v2@v2.0.1/helper/template) which is based on [`text/template`](https://pkg.go.dev/text/template).
```

Comment:
This explanation is quite hard to follow.  Can it (or somewhere else in the docs) be explained by example?

When you say `OIDC token` here, are you talking in particular about the OIDC id_token?

Reply:
Yes, this would be from the id_token if you're doing auth with an OIDC type role and not JWT:

https://github.com/openbao/openbao/blob/d393d9dae54dcb5234f654a3374dd9ec6e0d0be5/builtin/credential/jwt/path_oidc.go#L291

https://github.com/openbao/openbao/blob/d393d9dae54dcb5234f654a3374dd9ec6e0d0be5/builtin/credential/jwt/path_oidc.go#L332-L337

https://github.com/openbao/openbao/blob/d393d9dae54dcb5234f654a3374dd9ec6e0d0be5/builtin/credential/jwt/path_oidc.go#L436-L438

I'll work to revise the docs with some examples.

Reply:
@DrDaveD Pushed; I've also sent an email to schedule a meeting with everyone.